features/support/env.rb

```diff
@@ -2,6 +2,7 @@
 Coveralls.wear_merged!('rails')
 
 require 'cucumber/rails'
+require 'pry'
```

Comment:
You might want to remove this line here... If you're going to use `pry` make sure you add it to your Gemfile